internal/charmstore/server.go

```diff
@@ -103,6 +103,30 @@ type ServerParams struct {
 	// RunBlobStoreGC holds whether the server will run
 	// the blobstore garbage collector worker.
 	RunBlobStoreGC bool
+
+	// BlobStore holds the type of blobstore to use.
+	BlobStore string `yaml:"blobstore"`
+
+	// SwiftAuthURL holds the swift auth url, equivalent to OS_AUTH_URL.
+	SwiftAuthURL string `yaml:"swift-auth-url"`
```

Comment:
Perhaps just bunch the swift parameters into a single structure would be better.